Packs/QualysVulnerabilityManagement/Integrations/QualysVulnerabilityManagement/QualysVulnerabilityManagement.py

```diff
@@ -0,0 +1,1499 @@
+        'context_key': 'ID',
+    },
+    'qualys-vulnerability-list': {
+        'context_prefix': 'Qualys.Vulnerability.List',
+        'context_key': 'QID',
+    },
+    'qualys-group-list': {
+        'context_prefix': 'Qualys.AssetGroup',
+        'context_key': 'ID',
+    },
+    'qualys-vm-scan-fetch': {
+        'context_prefix': 'Qualys.VM',
+        'context_key': 'QID',
+    },
+    'qualys-pc-scan-fetch': {
+        'context_prefix': 'Qualys.PC',
+        'context_key': 'TITLE'
+    },
+    'qualys-report-cancel': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-report-delete': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-scorecard-launch': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-vm-scan-launch': {
+        'context_prefix': 'Qualys.Report.VM.Launched',
+        'context_key': 'KEY',
+    },
+    'qualys-vm-scan-action': {
+        'context_prefix': '',
+        'context_key': 'ID',
+    },
+    'qualys-pc-scan-launch': {
+        'context_prefix': 'Qualys.Scan',
+        'context_key': 'ID',
+    },
+    'qualys-pc-scan-manage': {
+        'context_prefix': 'Qualys.Scan',
+        'context_key': 'scan_ref',
+    },
+    'qualys-ip-add': {
+        'context_prefix': 'Qualys.IP.Add',
+        'context_key': 'IP',
+    },
+    'qualys-ip-update': {
+        'context_prefix': 'Qualys.IP.Update',
+        'context_key': 'IP',
+    },
+    'qualys-host-excluded-manage': {
+        'context_prefix': 'Qualys.Endpoint',
+        'context_key': 'KEY',
+    },
+    'qualys-scheduled-report-launch': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-report-launch-map': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-report-launch-scan-based-findings': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-report-launch-host-based-findings': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-report-launch-patch': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-report-launch-remediation': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-report-launch-compliance': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-report-launch-compliance-policy': {
+        'context_prefix': 'Qualys.Report',
+        'context_key': 'ID',
+    },
+    'qualys-virtual-host-manage': {
+        'context_prefix': 'Qualys.VirtualEndpoint',
+        'context_key': 'DATETIME',
+    },
+}
+
+# Information about the API request of the commands
+COMMANDS_API_DATA: Dict[str, Dict[str, str]] = {
+    'qualys-report-list': {
+        'api_route': API_SUFFIX + '/report/?action=list',
```

Comment:
About 1, it cannot be used as part of the client's base url because there are two possible API suffixes, that's why I added it this way. Then should I add use the f-strings as you suggested?

About 2, those list commands always use the action=list, it cannot run without it.
I think that the current way is the most elegant. Another option is to explicitly add action=list to the build_args_dict function specifically for list commands, another approach is to ask the user for adding action=list which can be confusing IMO. What do you think?